docs/tutorials/ci-cd.md

```diff
@@ -81,4 +81,19 @@ jobs:
 1. The `install` command will install the latest Flox version. You can change
    the `channel` and `version` option which allows you to select excatly which
    version of Flox to install.
-2. The `activate` command runs a command in the context of a Flox environment.
\ No newline at end of file
+2. The `activate` command runs a command in the context of a Flox environment.
+
+
+## GitLab
+
```

Comment:
could be nice to have a small amount of explainer of what you're going to do in GitLab (even if it's just "similar to GitHub, we are going to setup ci.yaml")